lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential_type.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+#
+# (c) 2018, Adrien Fleury <fleu42@gmail.com>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_credential_type
+author: "Adrien Fleury (@fleu42)"
+version_added: "2.7"
+short_description: >
```

Comment:
`short_description: Create, update, or destroy custom Ansible Tower credential type`

Reply:
This will fix the CI issue

Reply:
Thanks for checking. As it was valid YAML I didn't think it could fail the CI.

Reply:
Hi a strange corner case in `validate-modules` which are trying to work out how to fix.